uiplib/scheduler.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, offline, pics_folder, timeout, website, count, service):
         self.directory = pics_folder
         self.count = count
         self.service = service
+        self.Wallpaper = Wallpaper()
```

Comment:
maybe have just one wallpaper instance in main and pass it around to scheduler and GUI ? so that we don't conflict?